internal/k8s/image.go

```diff
@@ -49,6 +50,29 @@ func InjectImageDigest(entity K8sEntity, injectRef reference.Named, policy v1.Pu
 		return K8sEntity{}, false, fmt.Errorf("INTERNAL TILT ERROR: Cannot set PullNever with digest")
 	}
 
+	replaced := false
+	for _, f := range []func(K8sEntity, reference.Named) (K8sEntity, bool, error){
+		func(entity K8sEntity, named reference.Named) (K8sEntity, bool, error) {
+			return injectImageDigestInContainers(entity, named, policy)
+		},
+		injectImageDigestInEnvVars,
```

Comment:
this...seems like a lot of indirection to avoid an extra "err != nil" block

Reply:
well, two extra ```		if err != nil {
			return K8sEntity{}, false, err
		}
		if r {
			replaced = true
		}``` blocks, but fair enough